lib/her/model/orm.rb

```diff
@@ -269,9 +267,9 @@ def build(attributes = {})
           resource
         end
 
-        private
         # @private
         def blank_relation
+          @blank_relation ||= superclass.blank_relation.clone.tap { |r| r.parent = self } if superclass.respond_to?(:blank_relation)
```

Comment:
This now pulls the blank relation of of the superclass, similar to the way `default_scope` is inherited.

The one API change to `Relation` was to add a writer for `parent` so it could be updated from this context.